csrc/quantization/cutlass_w8a8/scaled_mm_entry.cu

```diff
@@ -221,18 +223,9 @@ void get_cutlass_moe_mm_data(
     const int64_t num_experts, const int64_t n, const int64_t k) {
   // This function currently gets compiled only if we have a valid cutlass moe
   // mm to run it for.
-  int32_t version_num = get_sm_version_num();
-#if defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90
```

Comment:
IIUC, we still need to have a check here, but it can be more flexible to allow for blackwell in addition to hopper